src/datachain/lib/dc.py

```diff
@@ -1943,6 +1943,7 @@ def from_csv(
             path : Storage URI with directory. URI must start with storage prefix such
                 as `s3://`, `gs://`, `az://` or "file:///".
             delimiter : Character for delimiting columns.
+                Leaving for backwards compatibility.
```

Comment:
How about "Takes precedence if also specified in `parse_options_config`"?

(I think it's fine to keep both, delimiter it probably commonly used and it's fine to keep it as an explicit option as a syntax sugar (or we can review it later)).